desktop-src/SecAuthZ/app-isolation-packaging-with-vs.md

```diff
@@ -44,32 +44,30 @@ Go to *Project -> Manage NuGet Packages* to install *Microsoft.Windows.SDK.Build
 
 :::image type="content" source="images/app-isolation/vs/07-SDK-BuildTools-Info.png" alt-text="A screenshot showing the info for the Build Tools package on the NuGet screen in Visual Studio":::
 
-## Step 4 - Edit the Packaging.appxmanifest file
+## Step 4 - Edit the Packaging.appxmanifest and project files
 
 In the manifest file, the following changes will need to be made:
 
 **Note**: Isolated win32 applications are not compatible with other application types within the same package.
 
-- Add `xmlns:previewsecurity2="http://schemas.microsoft.com/appx/manifest/preview/windows10/security/2"`
-to the `<Package>` element if it's not there already.
-  - Add `previewsecurity2` to `IgnorableNamespaces` at the end of the `<Package>` element.
-- Add `xmlns:uap10="http://schemas.microsoft.com/appx/manifest/uap/windows10/10"` to the `<Package>` element
-if it's not there already.
-  - Add `uap10` to `IgnorableNamespaces` at the end of the `<Package>` element.
+- Add `xmlns:uap18="http://schemas.microsoft.com/appx/manifest/uap/windows10/18"` to the `<Package>` element if it's not there already.
+  - Add `UAP18` to `IgnorableNamespaces` at the end of the `<Package>` element.
 - In `<Dependencies>` change `TargetDeviceFamily` to
-`<TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.26100.0" MaxVersionTested="10.0.26100.0" />`.
+`<TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.26100.0" MaxVersionTested="10.0.26226.0" />`.
 
   > [!NOTE]
   > Not all features are available in the minimum build, check out the [release notes](app-isolation-release-notes.md) for more detailed information.
 
-- In `<Application>` replace any existing entrypoint/trustlevel/runtimebehavior with `uap10:TrustLevel="appContainer" previewsecurity2:RuntimeBehavior="appSilo"`.
+- In `<Application>` replace any existing entrypoint/trustlevel/runtimebehavior with `EntryPoint="Windows.FullTrustApplication"` `uap18:EntryPoint="Isolated.App"` `uap18:TrustLevel="appContainer" uap18:RuntimeBehavior="appSilo"`.
```

Comment:
We should note that this configures the app as a "Single Package" and will run as a win32 where isolatedwin32 isn't supported